src/admin-bro-options.interface.ts

```diff
@@ -235,6 +235,11 @@ export interface AdminBroOptions {
    * internationalization in AdminBro works.
    */
   locale?: Locale;
+
+  /**
+   * The file path to babel config file or json object of babel config.
+   */
+  babelConfig?: string | Record<string, any> | Function;
```

Comment:
I would nest this under `bundler` options:

```
bundler: {
  babelConfig: string | Record<string, any> | Function;
}
```

and by writing `file path to babel config` you mean babel.rc?

Reply:
it would be good to provide some simple example in the comments (with @exmple tag)

Reply:
It might not be .babelrc, cause if your .babelrc contains some configurations which conflict with default rollup babel config(ex. babelHelpers: 'bundle' ), bundle will fail.
So it will be good if we has a config file which should only for rollup babel config.